projects/github.com/linux-test-project/lcov/package.yml

```diff
@@ -0,0 +1,46 @@
+distributable:
+  url: https://github.com/linux-test-project/lcov/releases/download/v{{ version.major }}.{{ version.minor }}/lcov-{{ version.major }}.{{ version.minor }}.tar.gz
```

Comment:
`v{{ version.major }}.{{ version.minor }}` is either `v{{ version.marketing }}` (if you're specifically dropping the patch level, or `v{{ version.raw }}` if you're dealing with a project that might have 2- or 3-digit versions.